File: AngularApp/src/app/models/ContaContabil.ts
import { Empresa } from "./Empresa";
import { GrupoConta } from "./GrupoConta";
import { Lancamento } from "./Lancamento";

export interface ContaContabil{
     id?: number;
     descricao: string;
     grupoContaId: number;
     grupoConta: GrupoConta;
     empresa: Empresa;
     empresaId: number;
     lancamentos: Lancamento[];
}

File: ProjetoAspNetCoreApp/ClienteProduto.Domain/Entities/ContaContabil.cs
namespace ClienteProjeto.Domain.Entities;

public class ContaContabil : Entity
{

    public string Descricao { get; set; }
    public GrupoConta GrupoConta { get; set; }
    public Empresa Empresa { get; set; }
    public ICollection<Lancamento> Lancamentos { get; set; }

    #region "EF"

    public int EmpresaId { get; set; }

    public int GrupoContaId { get; set; }

    #endregion
}
